spec/computer_spec.rb

```diff
@@ -1,73 +1,75 @@
 require 'spec_helper'
-require 'computer'
+
 require 'board'
+require 'player'
+require 'computer'
+require 'marks'
 
 RSpec.describe Computer do
   let (:input) { StringIO.new("1\n") }
   let (:output) { StringIO.new }
-  let (:ui) { CliInterface.new(input, output, "spec/lang/") }
```

Comment:
Yeah - I think it's possible to extract this part to another branch with Git but at the moment I don't know how to do it